### DISCUSSION TITLE
const me

### DISCUSSION THREAD

**Discussion by Policing2024** [SubtitleEdit/subtitleedit#8816] [Q&A] [answered]
whisper const me
commandline please

**Comment by niksedk** [comment, marked as answer]
Please always explain as much as possible when added new issues/discussions...

If you click on the "Advanced" button in the SE whisper window you can see the parameters:
![image](https://github.com/user-attachments/assets/dfb6bb8d-1869-4a3d-8a17-3d98a4afa3bf)

You can get an actual command line by running cost-me via whisper i SE and the viewing the "whisper_log.txt" file.
Like:
`main.exe --language en --model "<some-path>\tiny.en.bin" --output-srt  "<some-path>\filenamewav"`

Read more here: https://github.com/Const-me/Whisper

**Comment by Policing2024** [comment]
SIR I TRIED THESE COMMANDS ON CLI ROOT PATH, BUT IT DIDN'T WORK PLS.

**Reply by niksedk** [reply]
Go to Const-me forum, thx

**Comment by Policing2024** [comment]
Sir
i try it, but this is the output (repeated words) pls

**Comment by Policing2024** [comment]
SIR,
CAN I USE CONST ME COMMANDLINE ON POWERSHELL?
